app/templates/components/ui-table/cell/cell-tickets.hbs

```diff
@@ -1,8 +1,9 @@
-{{#if this.record}}
+{{#if this.extraRecords}}
   <div class="ui list">
-    {{#each this.record as |ticket|}}
+    {{#each this.extraRecords.tickets as |ticket|}}
       <div class="item">{{ticket.name}} ({{add ticket.orderStatistics.tickets.completed ticket.orderStatistics.tickets.placed}}/{{ticket.quantity}}) <span class="item muted text">[{{ticket.type}}]</span></div>
```

Comment:
make sure that this component is not used at any other place bcz u changed ```this.record``` -> ```this.extraRecords```.
This may break at other place

Reply:
then should I revert it back to normal as sorting is not required yet.

Reply:
Sorting is definitely desired.